src/index.js

```diff
@@ -1,4 +1 @@
-// Testing bundle works with webex and component libraries
-import webex from 'webex';
-import WebexSDKAdapter from '@webex/sdk-component-adapter';
```

Comment:
Why are this changes here? I think you need to rebase your branch from master again